### PR TITLE
[Integration Test][ome_application_certificate] - Added certification_chain

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by Saksham-Nautiyal — APPROVED** [review]

File: tests/integration/targets/ome_application_certificate/tests/invalid.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: hostname, username
+         and password.'
+  when: hostname is not defined or username is not defined or password
+         is not defined
+
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_certificate/tests/invalid.yaml:12]: Cover all other common negative scenarios. Invalid username, password, port etc.

File: tests/integration/targets/ome_application_certificate/tests/invalid_certificate_chain.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_certificate/tests/invalid_certificate_chain.yaml]: This has only one task. Do we need a separate test case for this?

File: tests/integration/targets/ome_application_certificate/tests/invalid_san.yaml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        that:
+          - invalid_san_m_char.failed
+          - not invalid_san_m_char.changed
+          - 'invalid_san_m_char.msg == "HTTP Error 400: 400"'
+          - invalid_san_m_char.error_info.error['@Message.ExtendedInfo'][0]
+            .Message == "Unable to complete the request because the input
+            value for Subject Alternative Name(s) is missing or an invalid
+            value is entered."
+
+    #   TITLE: Defect- JIT-277402 - Ansible(8.3.0)-[Latent]
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_certificate/tests/invalid_san.yaml]: This defect is fixed. Check this scenario and uncomment this step

File: tests/integration/targets/ome_application_certificate/tests/invalid.yaml (hunk 1 above)
Comment (ShivamSh3) [reply tests/integration/targets/ome_application_certificate/tests/invalid.yaml:12]: done

File: tests/integration/targets/ome_application_certificate/tests/invalid_san.yaml (hunk 3 above)
Comment (ShivamSh3) [reply tests/integration/targets/ome_application_certificate/tests/invalid_san.yaml]: done

File: tests/integration/targets/ome_application_certificate/tests/invalid_certificate_chain.yaml (hunk 2 above)
Comment (ShivamSh3) [reply tests/integration/targets/ome_application_certificate/tests/invalid_certificate_chain.yaml]: done

**Comment by ShivamSh3** [comment]
image

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: tests/integration/targets/ome_application_certificate/vars/main.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (rajshekarp87) [inline tests/integration/targets/ome_application_certificate/vars/main.yaml]: Move all this to inventory.networking and don't provide any values

Comment (ShivamSh3) [reply tests/integration/targets/ome_application_certificate/vars/main.yaml]: done

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Pull request merged**